### ISSUE TITLE
[FxA] The "Favorite" button does not have any tooltip

### ISSUE THREAD

**Issue by SoftVision-CosminMuntean** [mozilla-services/screenshots#5020]
[Affected versions]:
- Nightly 64.0a1
- Screenshots 34.0.0 dev

[Affected Platforms]:
- All Windows
- All Mac
- All Linux

[Prerequisites]:
- The "xpinstall.signatures.required" boolean pref is set to "false".
- The "extensions.legacy.enabled" boolean pref is set tot "true".
- Have the Screenshots 34.0.0 dev version installed from [here](https://screenshots.dev.mozaws.net/homepage/install-test-local.html), on a new clean profile. 

[Steps to reproduce]:
1. Open the Firefox browser with the profile from prerequisites and navigate to any website.
2. Save a selection on the page.
3. Hover the "Favorite" button and observe the behavior.

[Expected result]:
- A specific tooltip is displayed.

[Actual results]:
- No tooltip is displayed.

[Notes]:
- The issue is also reproducible if you are logged in with FxA. 
- Here is a screen recording of the issue:
![favorite](https://user-images.githubusercontent.com/19226534/46669651-b1227500-cbd8-11e8-8b22-7e42af3f1361.gif)

**Comment by johngruen** [comment]
#4994 may have regressed this

**Comment by johngruen** [comment]
@stasm any thoughts here?

**Comment by punamdahiya** [comment]
I am seeing other strings such as shotPageDownloadShot doesn't have the tooltip picked from server.ftl  @SoftVision-CosminMuntean can you please verify if it's showing up in languages other than en-us. We can update title of the bug accordingly. Thanks!

**Comment by SoftVision-CosminMuntean** [comment]
@punamdahiya the "Favorite" tooltip is not displayed in any languages. But, during testing this I have observed that the tooltips, buttons, and strings are not localized in the selected language. Are displayed in English. I have logged this issue in #5022

**Comment by punamdahiya** [comment]
Should be fixed with #4944 revert

**Comment by SoftVision-CosminMuntean** [comment]
I have verified this issue on latest Screenshots 34.0.0 dev version and is no longer reproducible. Tested on Windows 7 x64, Mac 10.13 and Arch Linux 4.12.
